### DISCUSSION TITLE
Feature: go1.20 standard log support

### DISCUSSION THREAD

**Discussion by godcong** [ent/ent#3186] [General]
- [x] I have searched the [issues](https://github.com/ent/ent/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

This is not an ent issue.
This is an ent enhancement proposal.
In the past few years, go's logging capabilities have been very weak (arguably non-existent).
But now `go 1.20` has enhanced logging.
I hope to support it in the future.

**Comment by giautm** [comment]
@godcong can you share a link about it. I can't find any thing yet.

**Comment by godcong** [comment]
@giautm 
I'm sorry, but I've been locked up at home for a long time because of the covid-19.

Information about the log package is as follows:
`https://github.com/golang/exp/tree/master/slog`
=>
`golang.org/x/exp/slog`

I hope that with the 1.20 Release standardized `log` support will be introduced.

I have used many open source packages in some of my past projects.
These packages all use their own logging.
I would like to collect these log messages as uniformly as possible.

The introduction of 1.20 `slog` is a good solution to my problem above.
